src/axios/sms-clients/interview-client.ts

```diff
@@ -20,32 +21,47 @@ export const interviewClient = {
 
     fetchPage: (pageNumber?: number, pageSize?: number, orderBy = 'id', direction = 'ASC',
         associateEmail = 'associateEmail', managerEmail = 'managerEmail',
-        place = "placeName", clientName = "clientName", staging = 'stagingOff') => {
+        place = "placeName", clientName = "clientName", /*staging = 'stagingOff',*/ input = 'associateInput', feedback = 'feedback') => {
         const currentUser = store.getState().managementState.auth.currentUser;
         const roles = currentUser.roles
+        console.log(roles);
         const email = currentUser.email
+        console.log(email);
         const isAdmin = (roles.includes(cognitoRoles.ADMIN) || roles.includes(cognitoRoles.STAGING_MANAGER) || roles.includes(cognitoRoles.TRAINER));
 
         let url = interviewContext;
-        url += '/page'
-        if (!isAdmin) url += 's'
-        url += '?orderBy=' + orderBy + '&direction=' + direction;
-        if (pageNumber) {
-            url += '&pageNumber=' + pageNumber;
-        }
-        if (pageSize) {
-            url += '&pageSize=' + pageSize;
-        }
-        if (!isAdmin)
-            url += '&email=' + email;
-        url += '&associateEmail=' + associateEmail;
-        url += '&managerEmail=' + managerEmail;
-        url += '&place=' + place;
-        url += '&clientName=' + clientName;
-        url += '&staging=' + staging;
-
-        return smsClient.get(url);
-    },
+
+    url += '/page'
+
+    // an associate user, uses pages endpoint instead of page
+    if(!isAdmin) { url += 's' }
```

Comment:
This bit of code is a bit hard to follow due to the formatting. It should probably also be abstracted to its own method for building the query parameters.